FAQ.md

```diff
@@ -35,7 +37,9 @@
     *   `getting-started`: issue or pull request is suitable for someone new to GitHub and/or this project
     *   `help-wanted`: a question or request for assistance
     *   `left-as-was`: marks an issue closed because the item in question will be left as it was
```

Comment:
On 2015-02-16 11:09 PM, Carol Willing wrote:
>
> In FAQ.md 
> <https://github.com/swcarpentry/lesson-template/pull/175#discussion_r24791326>:
>
> > @@ -35,7 +37,9 @@
> >      *   `getting-started`: issue or pull request is suitable for someone new to GitHub and/or this project
> >      *   `help-wanted`: a question or request for assistance
> >      *   `left-as-was`: marks an issue closed because the item in question will be left as it was
>
> |left-as-was| like |duplicate| is better handled in the closing 
> remarks of a pull request. Both are reasons for closing an issue or 
> PR. I suggest removing.
>
Seems reasonable to me.